source/_posts/2021-07-07-release-20217.markdown

```diff
@@ -257,8 +248,7 @@ noteworthy changes this release:
 - If you have a SmartTube, you can now reset your reminders, thanks [@mdz]!
 - KNX had quite a lot of updates and added support for XY-color lights,
   thanks [@farmio].
-- [@OttoWinter] added support for presets, custom presets and custom fan modes
-  for climate controls in ESPHome. Awesome!
+- [@OttoWinter] added support for presets, custom presets, and custom fan modes for climate controls in ESPHome. Awesome!
```

Comment:
This is adding an oxford comma, which we don't use. Please revert this change.

Reply:
ok